misc/docerina/src/main/java/org/ballerinalang/docgen/docs/BallerinaDocGenerator.java

```diff
@@ -120,13 +120,13 @@ public static void mergeApiDocs(String apiDocsRoot)  {
                         String htmlData = "<!DOCTYPE html>\n" +
                                 "<html>\n" +
                                 "<head>\n" +
-                                "\t<meta http-equiv=\"refresh\" content=\"0; URL=../index.html#/" +
+                                "\t<meta http-equiv=\"refresh\" content=\"0; URL=../index.html#/ballerina/" +
```

Comment:
Can there be `ballerinax` packages?

Reply:
Added the orgname to links with the new commit.